src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer start working");

```

Comment:
The message in the 'doWork' method should indicate the type of Machine more informatively. Consider specifying that a Bulldozer is working, for example: 'Bulldozer is starting its work.'